landscape/client/manager/aptsources.py

```diff
@@ -140,10 +140,15 @@ def _handle_sources(self, ignored, sources):
             "manage_sources_list_d",
             True,
         )
+
         if manage_sources_list_d not in FALSE_VALUES:
-            filenames = glob.glob(os.path.join(self.SOURCES_LIST_D, "*.list"))
-            for filename in filenames:
-                shutil.move(filename, f"{filename}.save")
+            patterns = ["*.list", "*.sources"]
```

Comment:
since this is a constant value I think you should define it in a top-level or class-level constant rather than building the list here, along with a comment explaining its purpose, if it isn't obvious from the variable name.